core/config/toml/types.go

```diff
@@ -1301,3 +1303,29 @@ func (t *ThresholdKeyShareSecrets) validateMerge(f *ThresholdKeyShareSecrets) (e
 
 	return err
 }
+
+type Tracing struct {
```

Comment:
Is there anything worth validating in a `ValidateConfig` method?

Reply:
Potentially the `CollectorTarget`, though any issues with the target address will be caught here: https://github.com/smartcontractkit/chainlink-relay/pull/179/files#diff-b52b12bf1ea0e82a0448f1c6cbe6e1bcb8bda2f10b50050809b0c199d0a7e79fR63

Reply:
Don't we just log those errors and move on though?

Regardless, we aim to as much validation up front as we can, so that we can produce coherent organized feedback on the configuration file itself, rather than obscure internal issues later on that don't make sense to node ops.

Reply:
It also allows them to pre-validate before running the node.

Reply:
Agreed -  it's a better experience to validate up front